migration/migrator/migrations/course/20220612122726_gradeable_specific_anon_ids.py

```diff
@@ -0,0 +1,34 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("CREATE TABLE IF NOT EXISTS gradeable_anon (user_id character varying NOT NULL, g_id character varying(255) NOT NULL, anon_id character varying(255));")
```

Comment:
Can you add foreign keys for the user id and gradeable id

Reply:
Done